runpod_worker/client_example.py

```
@@ -0,0 +1,287 @@
+#!/usr/bin/env python3
+"""
+CharForgex RunPod Client Example
+Demonstrates how to interact with the CharForgex RunPod worker
+"""
+
+import requests
+import base64
+import time
+import json
```

Ruff (0.12.2)

`json` imported but unused

Remove unused import: `json`

(F401)